.deploy/Dockerfile

```
@@ -7,7 +7,22 @@
+ARG DOPPLER_CONFIG="prod"
+ARG DJANGO_CONFIGURATION="Production"
+
+WORKDIR /src/app
+EXPOSE 9090
+
+RUN useradd --shell /bin/bash --create-home api
+COPY --chown=api:api ./requirements/base.txt ./requirements.txt
+COPY --chown=api:api  Taskfile /src/Taskfile
+
+COPY --chown=api:api  thedozens/ /src/app/
+RUN python -m pip install -r ./requirements.txt
```

Hadolint (2.12.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)